app/assets/stylesheets/hackney_work_order.scss

```diff
@@ -46,7 +46,8 @@
 .main-section-box {
   min-height: 252px;
 }
-.hackney-main-section-info p, ul {
+.hackney-main-section-info p,
+.hackney-main-section-info ul {
```

Comment:
why do it this way?

Reply:
The line I removed was matching all `p` inside a `.hackney-main-section-info` and every `ul` (regardless of being inside `.hackney-main-section-info`).

I just prepended the class to the `ul`, which I suppose was the intended effect, to fix it interfering with the new warning labels I did.

Reply:
*warning labels for tenure type, tmo and cautionary contact